setup.py

```diff
@@ -18,13 +18,13 @@
     # system-verilog parser
     'svinst==0.1.5',
     # magma ecosystem dependencies
-    'fault==3.0.36',
-    'magma-lang==2.1.17',
-    'coreir==2.0.120',
-    'mantle==2.0.10',
-    'hwtypes==1.4.3',
-    'ast_tools==0.0.30',
-    'kratos==0.0.31.1',
+#    'fault==3.0.36',
```

Comment:
I think these were probably commented out accidentally and should be uncommented.

Reply:
They aren't commented out accidentally. The server doesn't support these libraries and this was the fast solution to start physical design. They were accidentally committed however. 

(Tester is commented out because there is no magma/fault available).

Reply:
OK, thanks for putting them back.  When there is more time post-tapeout, it would be great to find out more about this issue.  I imagine that the **magma** developers would like to support our design servers.